src/stores/spaces/SpaceStore.ts

```diff
@@ -99,9 +100,11 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
     private parentMap = new EnhancedMap<string, Set<string>>();
     // Map from SpaceKey to SpaceNotificationState instance representing that space
     private notificationStateMap = new Map<SpaceKey, SpaceNotificationState>();
-    // Map from space key to Set of room IDs that should be shown as part of that space's filter
+    // Map from SpaceKey to Set of room IDs that are direct descendants of that space
     private spaceFilteredRooms = new Map<SpaceKey, Set<string>>(); // won't contain MetaSpace.People
-    // Map from space ID to Set of user IDs that should be shown as part of that space's filter
+    // Map from SpaceKey to Set of space keys that are direct descendants of that space
+    private childSpacesBySpace = new Map<SpaceKey, Set<SpaceKey>>(); // won't contain MetaSpace.People
```

Comment:
Well surely it won't contain any metaspaces, not just People, given metaspaces can't contain other spaces.